deeplake/core/vectorstore/vector_search/indra/query.py

```diff
@@ -26,14 +26,25 @@ def create_query_string(
         str: TQL representation of the query string.
     """
 
+    # tql_filter_str = tql_filter if tql_filter == "" else " where " + tql_filter
+    # tensor_list_str = ", ".join(tensor_list)
+    # order_str = "" if order is None else f" order by score {order}"
+    # distance_metric_str = (
+    #     "" if distance_metric is None else f", {distance_metric} as score"
+    # )
+
+    # return f"select * from (select {tensor_list_str}{distance_metric_str}{tql_filter_str}){order_str} limit {limit}"
+
+    ## TODO: DELETE THIS AND BRING BACK ABOVE IMPLEMNTATION
```

Comment:
@sounakr @khustup we need to delete this and bring back the implementation above when things are fixed in TQL, since the implementation above is verified to work with all the queries we need to support.